src/MaterialDesignThemes.Wpf/Converters/ClockItemIsCheckedConverter.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace MaterialDesignThemes.Wpf.Converters;
 
-internal class ClockItemIsCheckedConverter(Func<DateTime> currentTimeGetter, ClockDisplayMode displayMode, bool is24Hours) : IValueConverter
+public class ClockItemIsCheckedConverter(Func<DateTime> currentTimeGetter, ClockDisplayMode displayMode, bool is24Hours) : IValueConverter
```

Comment:
This one I think we can leave internal since its only usages were in code and not XAML.